eclipse-collections/src/main/java/org/eclipse/collections/impl/collection/mutable/AbstractMultiReaderMutableCollection.java

```diff
@@ -78,231 +79,139 @@
  */
 public abstract class AbstractMultiReaderMutableCollection<T> implements MutableCollection<T>
 {
-    protected abstract MutableCollection<T> getDelegate();
-
-    protected abstract ReadWriteLock getLock();
-
-    protected void acquireWriteLock()
-    {
-        this.getLock().writeLock().lock();
-    }
+    protected transient ReadWriteLock lock;
```

Comment:
do we still need this field?

Reply:
There are other parallel classes which still depend on the field.  I will get rid of it once I get a chance to refactor those classes as well.